### PR TITLE
Create minutes-2017-03-08.md

### PR DESCRIPTION
**Pull request by raynamharris**
minutes from today's meeting

### CODE REVIEW

File: minutes/minutes-2017-03-08.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+#### Minutes
+- Erin is not present. No update.
+- Erin will not sit in for every SC meeting.
+- there is a discussion about if the trainer can also do a demo session for the person who he taught; more info: https://github.com/swcarpentry/instructor-training/issues/392
+- there is a need for more info on certification process
+- There is an agenda 3 months forward where new trainers can fill in the slots when they’re available 
+- **Action**: Christina will have a discussion with Erin formalizing processes regarding training and certification (with Belinda)
+- **Action**:Rayna & Christina will have report back from trainer subcommittee for each meeting rather than have a instructor director’s update.
+
```
Comment (karinlag) [inline minutes/minutes-2017-03-08.md]: I think we also agreed on putting in a more formal document describing who should decide what re training, what can the committee decide on, what can Erin decide on, and what should be the purview of the SC(s).

**Comment by raynamharris** [comment]
@karinlag Please see the change and let me know if that's okay.

**Comment by ChristinaLK** [comment]
I think that's clear.

**Comment by karinlag** [comment]
@raynamharris looks good.

**Pull request merged**